<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>h264_source_local_audioClass</class>
 <widget class="QMainWindow" name="h264_source_local_audioClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>h264_source_local_audio</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="openUrl">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>96</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rtmp://gs.live.rgbvr.com/live/123456</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>拉流地址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="openUrl_2">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>147</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>rtmp://gs.push.rgbvr.com/live/654321</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>推流地址：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startStreamBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始拉流</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始推流</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="h264_source_local_audio.qrc"/>
 </resources>
 <connections/>
</ui>
